qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -929,6 +930,27 @@ def from_sparse_list(
         paulis = PauliList(labels)
         return SparsePauliOp(paulis, coeffs, copy=False)
 
+    @staticmethod
+    def from_sparse_observable(obs: SparseObservable) -> SparsePauliOp:
```

Comment:
Do you want to add a variant to the `__init__` method that handles this case too?  `quantum_info` types (for better or worse) check everything under the sun in their default constructors.

Reply:
I think we said we don't want to add this because it would give a very easy path to a potentially very expensive representation -- so it's after a "you-need-an-explicit-constructor"-wall 😄

Reply:
Ok sure, that's valid too.  I forgot we'd said that haha